packages/loader/container-loader/src/container.ts

```diff
@@ -669,6 +669,11 @@ export class Container
 		return this._clientId;
 	}
 
+	public get deltaConnectionMetadata() {
+		// Provide a copy instead of the actual object.
+		return { ...this._deltaManager.deltaConnectionMetadata };
+	}
```

Comment:
I think that's fine, but it feels like the ideal solution is
- Layers propagate events
- Only final layer (Container) accumulates state and exposes it. In other words, I'd expect DM not to have that method.

This greatly depends on how / when events are delivered (and ability to lose events through layers), of course.'

Just looking for simpler code (less lines :))

Reply:
Took this approach.